build.sbt

```diff
@@ -259,20 +257,9 @@ def multiScalaDirectories(root: File, scalaVersion: String) = {
   val base = root / "src" / "main"
   val result = mutable.ListBuffer.empty[File]
   val partialVersion = CrossVersion.partialVersion(scalaVersion)
-  if (isScala211(partialVersion)) {
-    result += base / "scala-2.11"
-  }
-  if (isScala212(partialVersion)) {
-    result += base / "scala-2.12"
-  }
-  if (isScala213(partialVersion)) {
-    result += base / "scala-2.13"
-  }
-  if (isScala023(partialVersion)) {
-    result += base / "scala-0.23"
-  }
-  if (isScala024(partialVersion)) {
-    result += base / "scala-0.24"
+  partialVersion.collect {
```

Comment:
This should simplify things and make it more future proof.

Reply:
Ah, I like this. Good idea 👍